content/conduct.md

```diff
@@ -3,8 +3,24 @@ title: Code of Conduct
 type: page
 ---
 
-# Code of conduct
+# Code of Conduct
+
+We've found that code reading clubs work best when you involve people of different backgrounds and levels of experience. People read code differently and we can learn a lot from each other! It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
```

Comment:
```suggestion
We've found that code reading clubs work best when you involve people with different backgrounds and levels of experience. People read code differently and we can learn a lot from each other. It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
```

Reply:
Maybe reads better as with and one !

The tone and content is ace.

Reply:
I drafted this after watching a !!con talk so I was in a heightened exclamation-mark mood 😅